lib/TableGen/Operations.cpp

```diff
@@ -711,11 +711,11 @@ void BuilderMethod::emitDefinition(raw_ostream &out, FmtContext &fmt,
     out << tgfmt(R"(
       };
       $varArgInitializer
-      return ::llvm::cast<$_op>($_builder.CreateCall($fn, $args));
+      return ::llvm::cast<$_op>($_builder.CreateCall($fn, $args, instName));
     )",
                  &fmt);
   } else {
-    out << tgfmt("return ::llvm::cast<$_op>($_builder.CreateCall($fn));\n",
+    out << tgfmt("return ::llvm::cast<$_op>($_builder.CreateCall($fn, std::nullopt, instName));\n",
```

Comment:
Does the use of `std::nullopt` require an additional include, `<optional>`, in the generated dialect definition?

Reply:
It's the default argument in IRBuilder which pulls in the header

Reply:
You are right. Thanks.